packages/arb-token-bridge-ui/src/token-bridge-sdk/index.ts

```diff
@@ -1,19 +1,19 @@
-export * from './hooks/useArbTokenBridge'
-export * from './hooks/arbTokenBridge.types'
+export * from '../hooks/useArbTokenBridge'
+export * from '../hooks/arbTokenBridge.types'
```

Comment:
My only concern is why are `types` inside the hooks folder. If anyone has a better solution, pls lemme know 🥲.